about.md

```
@@ -1,22 +1,52 @@
+At Nirmata, I've developed core components of the **AI Platform Engineering Assistant** (launched at KubeCon North America 2025):
 
-Apart from the last three years of experience in DL, I am branching my skills into the world of Open-Source technologies such as Linux, Containers and Serverless Computing.
+- **[Kyverno MCP Server](https://github.com/nirmata/kyverno-mcp)** - A Model Context Protocol server that enables language models to generate and validate Kubernetes policies using Kyverno
+- **NCTL AI Agent** - A conversational CLI tool that transforms natural language into policy-as-code
+- **Multi-agent orchestration systems** for policy workflows that reduce what traditionally takes hours into minutes
 
-As an AI developer, I always faced the challenge of shipping my application due to the lack of underlying infrastructure and the missing approach of making my application accessible to a large user base (scaling my application without hiccups).
+## Technical Expertise
 
-In 2024, I aim to bring my skills in AI with Cloud Native technologies such as Containerd and Kubernetes to be able to serve my compute intensive apps to a wide audience.
+**Infrastructure & Cloud Native**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)